src/test/unit/service/listManipulation/opaPublicListService.test.ts

```diff
@@ -0,0 +1,48 @@
+import { expect } from 'chai';
+import fs from 'fs';
+import path from 'path';
+import { OpaPublicListService } from '../../../../main/service/listManipulation/OpaPublicListService';
+
+const opaPublicListService = new OpaPublicListService();
+
+describe('OPA Public List Service', () => {
+    const rawData = fs.readFileSync(path.resolve(__dirname, '../../mocks/opaPublicList.json'), 'utf-8');
+    const input = JSON.stringify(JSON.parse(rawData));
+
+    it('should format individual defendant name', async () => {
```

Comment:
Might be worth test defendant from organisation details as I can see you have logic in the service class for that. Just need to add that case in the mock test file